static/js/containers/App.js

```diff
@@ -36,7 +38,12 @@ class App extends React.Component {
   }
 
   loadData = async () => {
-    const { dispatch } = this.props
+    const { dispatch, location: { pathname } } = this.props
+
+    if (pathname === "/auth_required/") {
+      return
+    }
```

Comment:
what do you think about putting this check into `componentDidMount` (where we call `loadData`) instead of here? To me its just a little bit more clear to keep `loadData` a function which just loads some particular data, and decide whether or not we want to call it, instead of having logic inside of the `loadData` function to make that decision.